### PR TITLE
Change gateway fields to radio input fields. #658

### PR DESCRIPTION
**Pull request by pippinsplugins**
#658

### CODE REVIEW

File: templates/register.php [hunk 1]
```diff
@@ -104,13 +104,14 @@
 		$gateways = rcp_get_enabled_payment_gateways();
 		if( count( $gateways ) > 1 ) : $display = rcp_has_paid_levels() ? '' : ' style="display: none;"'; ?>
 			<fieldset class="rcp_gateways_fieldset">
+				<leged><?php _e( 'Choose Your Payment Method', 'rcp' ); ?></legend>
```
Comment (mindctrl) [inline templates/register.php]: misspelled `<legend>` here.

File: templates/register.php [hunk 2]
```diff
@@ -104,13 +104,14 @@
...
-						<?php foreach( $gateways as $key => $gateway ) : $recurring = rcp_gateway_supports( $key, 'recurring' ) ? 'yes' : 'no'; ?>
-							<option value="<?php echo esc_attr( $key ); ?>" data-supports-recurring="<?php echo esc_attr( $recurring ); ?>"><?php echo esc_html( $gateway ); ?></option>
-						<?php endforeach; ?>
-					</select>
-					<label for="rcp_gateway"><?php _e( 'Choose Your Payment Method', 'rcp' ); ?></label>
+					<?php foreach( $gateways as $key => $gateway ) : $recurring = rcp_gateway_supports( $key, 'recurring' ) ? 'yes' : 'no'; ?>
+						<label class="rcp_gateway_option_label">
+							<input name="rcp_gateway" type="radio" class="rcp_gateway_option_input" value="<?php echo esc_attr( $key ); ?>" data-supports-recurring="<?php echo esc_attr( $recurring ); ?>">
+							<?php echo esc_html( $gateway ); ?>
+						</labe>
```
Comment (mindctrl) [inline templates/register.php]: misspelled `</label>` here.

File: templates/register.php [hunk 3]
```diff
@@ -104,13 +104,14 @@
...
+					<?php endforeach; ?><br/>
```
Comment (mindctrl) [inline templates/register.php]: haven't tested yet, but curious why we need a `<br/>` here. would rather use CSS if possible.

Comment (pippinsplugins) [inline templates/register.php]: @mindctrl It was simply to avoid a CSS change but I'm fine with a CSS change instead.

**Comment by pippinsplugins** [comment]
Adjusted.

**Pull request merged**